docker-compose.yml

```diff
@@ -57,6 +57,7 @@ services:
       dockerfile: docker/Storage/Dockerfile
     volumes:
       - storage-data:/app/storage
+      - ./modyn/config/examples:/src/modyn/config/examples/
 #      - /mnt/datasets:/datasets
```

Comment:
why are these three changes are needed?

Reply:
I found it useful for debugging to be able to adjust the `modyn_config.yaml` and have it reflected in the container. I guess this mapping doesn't do any harm. If you don't find it useful we don't have to include it in the commit :)

Reply:
I think it takes more steps than just mapping the config: you need to restart the servers in the container. (which also may not always work currently because with the current impl, some other servers may just crash when failing to connect to the restarting ones...)

Also it might confuse others as we already copy the files into the container [here](https://github.com/eth-easl/modyn/blob/ed861c92cc05f7ab99b2694df4f1281234539639/docker/Base/Dockerfile#L4).

Anyway let's not do this in this commit

Reply:
Okay, I did a `docker exec` into the container and triggered the script there. In that context having a mapped file made sense. But I agree that it might not be useful in other situations, let's remove it!